### PR TITLE
feat: ui migration from html to react components

### PR DESCRIPTION
**Pull request by shobit1337**
Migrated old HTML screens to react components

### CODE REVIEW

File: src/components/Card/CategoryCard.jsx [hunk 1]
```diff
@@ -1,7 +1,15 @@
-import React from "react";
+import React from 'react';
 
 const CategoryCard = () => {
-  return <>Category Card</>;
+  return (
+    <a href='/pages/browse-products.html' className='category-card'>
```
Comment (znjs) [inline src/components/Card/CategoryCard.jsx]: Suggestion: Change this to a Router `Link` tag

File: src/components/Card/ProductCard.jsx [hunk 2]
```diff
@@ -1,7 +1,28 @@
-import React from "react";
+import React from 'react';
 
 const ProductCard = () => {
-  return <>Product Card</>;
+  return (
+    <a href='/pages/product.html' className='product-card'>
```
Comment (znjs) [inline src/components/Card/ProductCard.jsx]: Suggestion: Change this to a Router `Link` tag

File: src/components/Card/WishlistCard.jsx [hunk 3]
```diff
@@ -1,7 +1,41 @@
...
+            <span> Sale ends 3/4/2022 at 4:30 AM </span>
+          </div>
+        </div>
+
+        <div className='d-flex justify-between items-center'>
+          <i className='fab fa-windows'></i>
+          <div className='d-flex items-center gap-sm'>
+            <button className='link text-dark-lighter'>Remove</button>
+            <button className='btn btn-outlined btn-rounded btn-light btn-sm'>
+              ADD TO CART
```
Comment (znjs) [inline src/components/Card/WishlistCard.jsx]: Suggestion: Text can be Move to Cart

File: src/components/Footer/Footer.jsx [hunk 4]
```diff
@@ -1,7 +1,84 @@
...
+        <span>
+          <a href='/'>
+            <i className='fab fa-facebook-square'></i>
+          </a>
+          <a href='/'>
+            <i className='fab fa-twitter-square'></i>
+          </a>
+          <a href='/'>
+            <i className='fab fa-youtube-square'></i>
+          </a>
```
Comment (znjs) [inline src/components/Footer/Footer.jsx:16]: Suggestion: Can use some `margin-inline`

File: src/pages/wishlist/WishlistPage.jsx [hunk 5]
```diff
@@ -1,7 +1,20 @@
 import React from "react";
+import WishlistCard from "../../components/Card/WishlistCard";
```
Comment (znjs) [inline src/pages/wishlist/WishlistPage.jsx]: Suggestion: Can use barrel export

**Review by znjs — APPROVED** [review]
Clean code
Good naming conventions used
Nice file structuring

File: src/components/Navbar/Navbar.jsx [hunk 6]
```diff
@@ -1,7 +1,31 @@
...
-  return <>NavBar</>;
+  return (
+    <div className='sticky-header text-dark-light text-medium'>
+      <div className='input-text-group'>
+        <i className='fas fa-search'></i>
+        <input type='text' placeholder='Search' className='input-field' />
+      </div>
+      <div
+        className='d-flex justify-between items-center'
+        style={{ width: '100%' }}>
```
Comment (samflab) [inline src/components/Navbar/Navbar.jsx:13]: refrain from using inline-style

File: src/pages/auth/LoginPage.jsx [hunk 7]
```diff
@@ -1,7 +1,53 @@
...
+        <button className='btn btn-rounded text-light'>Sign In</button>
+        <span>
+          Forgot your password?
+          <a className='text-light' href='/pages/forgot-password.html'>
+            Reset
+          </a>
+        </span>
+        <span>
+          Don't have a Gamotore Account?
+          <a className='text-light' href='/pages/sign-up.html'>
```
Comment (samflab) [inline src/pages/auth/LoginPage.jsx]: don't forget to add the `Link` here instead of `a` tag

File: src/pages/product/ProductPage.jsx [hunk 8]
```diff
@@ -1,7 +1,82 @@
...
 
 const ProductPage = () => {
-  return <div>ProductPage</div>;
+  return (
+    <>
+      <h1 className='product-title border-top py-sm'>Cyberpunk 2077</h1>
+      {/* <div className='product-nav'>
+        <a href='#'>Overview</a>
+        <a href='#'>Achievements</a>
+      </div> */}
```
Comment (samflab) [inline src/pages/product/ProductPage.jsx]: you can remove commented code

**Review by samflab — APPROVED** [review]
Everything else looks good. I personally liked the design, its really cool and neat. The code and the file structure are readable.

Good work!

**Review by Mandhularajitha — APPROVED** [review]
refrain from using inline-style

File: src/pages/auth/LoginPage.jsx (hunk 7 above)
Comment (shobit1337) [reply src/pages/auth/LoginPage.jsx]: Yes, thank you for pointing it out

**Pull request merged**